docs/red_eyes_mainnet_tutorial.rst

```diff
@@ -0,0 +1,165 @@
+       "token_address": "0x255Aa6DF07540Cb5d3d297f0D0D4D84cb52bc8e6",
+       "balance": 2000,
+       "settle_timeout": 600
+   }
+
+At this point the specific value of the ``balance`` field isn't too important, since it's always possible to :ref:`deposit more tokens <topping-up-a-channel>` to a channel if need be.
+
+Successfully opening a channel returns the following information:
+
+.. sourcecode:: http
+
+   HTTP/1.1 201 CREATED
+   Content-Type: application/json
+
+   {
+       "channel_identifier": "0xfb43f382bbdbf209f854e14b74d183970e26ad5c1fd1b74a20f8f6bb653c1617",
+       "token_network_identifier": "0x3C158a20b47d9613DDb9409099Be186fC272421a",
+       "partner_address": "0x61C808D82A3Ac53231750daDc13c777b59310bD9",
+       "token_address": "0x255Aa6DF07540Cb5d3d297f0D0D4D84cb52bc8e6",
+       "balance": 2000,
+       "state": "opened",
+       "settle_timeout": 600,
+       "reveal_timeout": 10
+   }
+TODO: update token_network_identifier once it's known
+
+.. _doing-payments:
+
+Payments
+========
+Now that we have joined a token network, we can start making payments. For this, we need the address of the RDN token and the address of the recipient of the payment:
+
+.. http:example:: curl wget httpie python-requests
+
+    POST /api/1/payments/0x255aa6df07540cb5d3d297f0d0d4d84cb52bc8e6/0x61C808D82A3Ac53231750daDc13c777b59310bD9 HTTP/1.1
+    Host: localhost:5001
+    Content-Type: application/json
+
+    {
+        "amount": 42
+    }
+
+In this specific case the payment goes directly to one of our channel partners, since we have an open channel with ``0x61C808D82A3Ac53231750daDc13c777b59310bD9``. If we specify an address that we do not have a direct channel with, the Raiden Network finds a path to the target address and use mediated transfers to make a payment from our address to the target address.
+
+It's as simple as that to do payments using the Raiden Network. The first payment is done after just two API calls - one to join the token network and one to do the payment. The third call to open a direct channel is optional.
+
+Let's say we know someone with the address ``0x00014853D700AE1F39BA9dbAbdeC1c8683CF1b2A``, who is also part of the RDN token network. Even though we do not have a channel with this person it is as easy as above to send a payment. All we need is the address:
+
+.. http:example:: curl wget httpie python-requests
+
+    POST /api/1/payments/0x255aa6df07540cb5d3d297f0d0d4d84cb52bc8e6/0x00014853D700AE1F39BA9dbAbdeC1c8683CF1b2A HTTP/1.1
+    Host: localhost:5001
+    Content-Type: application/json
+
+    {
+        "amount": 73
+    }
+
+Just like this we can send payments to anyone who is part of the token network for the RDN token.
+
+.. _topping-up-a-channel:
+
+Depositing tokens
+=================
+If we are spending more tokens than we are receiving and hence depleting our channels, it it possible to "top up" channels. For this we need the token address and the partner address:
```

Comment:
I think it should be *deplete* without the *ing* here.

Reply:
if in doubt, drop the `ing`...

> If we spend more tokens than we receive and hence deplete our channels